corehq/apps/receiverwrapper/views.py

```diff
@@ -101,12 +101,14 @@ def _process_form(request, domain, app_id, user_id, authenticated,
             )
 
         app_id, build_id = get_app_and_build_ids(domain, app_id)
+        build_profile_id = request.META.get('buildProfileId')
```

Comment:
this needs to be updated once the mobile implementation is done, before merging it.